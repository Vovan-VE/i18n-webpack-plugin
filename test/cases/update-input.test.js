import fs from 'fs';
import processFile from '../cases.setup';

describe('update-input', () => {
  let extracted;
  const inputFileName = 'input.src.json';

  beforeAll(() => processFile('simple.code.js', { inputFileName })
    .then(({ file }) => {
      extracted = fs.readFileSync(file, 'utf8');
    }));

  it('should return merged with translations', () => {
    expect(extracted).toMatchSnapshot();
  });
});
