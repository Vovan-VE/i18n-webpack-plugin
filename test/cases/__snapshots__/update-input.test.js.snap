// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`update-input should return merged with translations 1`] = `
"{
  \\"category1\\": {
    \\"Message 1\\": \\"Translation 1\\",
    \\"Message 2\\": \\"\\"
  },
  \\"category2\\": {
    \\"Message 3\\": \\"\\"
  }
}"
`;
